form.yml

```diff
@@ -0,0 +1,341 @@
+netweaver:
+  $name: Netweaver
+  $type: group
+  
+  install_packages:
+    $name: Install required packages
+    $type: boolean
+    $default: true
+    $help: Install all the required packages to install SAP Netweaver
+  
+  virtual_addresses:
+    $name: Virtual Addresses
+    $type: edit-group
+    $help: Define the IP addresses and hostnames for machines used in SAP Netweaver installation. These machines will be part of the HA cluster
```

Comment:
I think we could even provide an example

Reply:
When filling this virtual addresses section in the form, there is IP address and hostname section which is quite straightforward, but I will add an example as you say